src/components/signup/Signin.jsx

```diff
@@ -1,3 +1,92 @@
-export function Signin() {
-  return <></>;
+import { useState } from "react";
+import "./signup.css";
+export function Signin({ setIsSignUp }) {
+  const [showPassword, setShowPassword] = useState(false);
+  const [loginForm, setLoginForm] = useState({
+    email: "",
+    password: "",
+  });
+
+  const changeHandler = event => {
+    event.preventDefault();
+    setLoginForm(() => ({
+      ...loginForm,
+      [event.target.name]: event.target.value,
```

Comment:
You can destructure the event to write smaller bits of code

Reply:
Made the change to use destructure. Thanks!